Module/Rule/Convert/Data.McAfee.ps1

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+<#
+    Instructions:  Use this file to add/update/delete regsitry expressions that are used accross 
+    multiple technologies files that are considered commonly used.  Enure expressions are listed
+    from MOST Restrive to LEAST Restrictive, similar to exception handling.  Also, ensure only
+    UNIQUE Keys are used in each hashtable to orevent errors and conflicts.
+#>
+
+$global:SingleLineRegistryValueName += [ordered]@{
```

Comment:
Without testing this myself, due to the selection similarities, there may be some efficiencies we could do.  Like the select in line 14 and 18, could it work like this:

'(?<=If the value (of|for)\s)(\w+)'